src/routes/ConsortiumManager/settings/authorization-roles/AuthorizationRolesViewPage.test.js

```diff
@@ -72,10 +73,6 @@ describe('AuthorizationRolesViewPage', () => {
     useAuthorizationRoles.mockImplementation(() => ({ roles: mockRoles }));
   });
 
-  afterAll(() => {
-    jest.clearAllMocks();
-  });
```

Comment:
What is the reason for removing the mock clearing?

Reply:
I don't see a reason to have it :) What the reason to have `clearAllMocks` in `afterAll`?